gnomad_qc/v4/annotations/vrs_annotation_batch.py

```diff
@@ -0,0 +1,523 @@
+"""
+This is a batch script which adds VRS IDs to a Hail Table by creating sharded VCFs, running a vrs-annotation script on each shard, and merging the results into the original Hail Table.
+
+Advise to run from a backend (eg. hailctl batch submit) to avoid losing progress
+ in case of disconnection.
+
+Example command to use `hailctl batch submit`:
+hailctl batch submit \
+--image-name us-central1-docker.pkg.dev/broad-mpg-gnomad/images/vrs084 \
+gnomad_qc/gnomad_qc/v4/annotations/vrs_annotation_batch.py \
+-- \
```

Comment:
is this an extra line?

Reply:
this is a bug for hailctl batch submit, they are changing it to match hailctl dataproc submit. We needed to add `--` for arguments from our script.

Reply:
I added a TODO.